google/cloud/bigtable/mutation_batcher.cc

```diff
@@ -22,8 +22,10 @@ namespace cloud {
 namespace bigtable {
 inline namespace BIGTABLE_CLIENT_NS {
 
-// Cloud Bigtable doesn't accept more than this.
+// Cloud Bigtable doesn't accept more than this in a single request.
 auto constexpr kBigtableMutationLimit = 100000;
+// Maximum mutations that can be outstanding in the Cloud Bigtable front end.
+auto constexpr kBigtableOutstandingMutationLimit = 300000;
```

Comment:
There is something odd about tracking in one process a limit that is system-wide... I may only send 100,000 mutations, but if you sent 250,000 I am affected too.

I guess I can prevent hurting myself, but this seems brittle.